client/src/components/MapView/MapView.tsx

```diff
@@ -78,7 +79,7 @@ export function MapView({
         }}
         // it's unfortunate that you have to list these layers here.
         // maybe there is a way around it: https://github.com/visgl/react-map-gl/discussions/2343
-        interactiveLayerIds={['regular-stop', 'area-stop', 'group-stop', 'parking-vertex', 'vertex', 'edge', 'link']}
+        interactiveLayerIds={interactiveLayerIds}
```

Comment:
Fantastic! I was waiting for someone to come up with a solution for this.

Reply:
You could remove the coment above it.

Reply:
It has been removed :bow: